### PR TITLE
Make batch size configurable

### PR DESCRIPTION
**Pull request by ziggythehamster**
Batch size now officially defaults to 500 in a non-Chef-managed agent, so reflect that new default in Chef. In addition, it's now configurable, should a customer need to raise this limit.

### CODE REVIEW

**Review by rclagett — APPROVED** [review]
This is great. Thank you @johvet and @ziggythehamster !

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Releasing
 ### What we do now
 1. Ensure that version has been bumped with semantic versioning (see Makefile)
 
-1. Ensure that the CHNAGELOG has been updated
+1. Ensure that the CHANGELOG has been updated
```
Comment (majormoses) [inline CONTRIBUTING.md:77]: :man_facepalming: thanks for fixing my mistake.

**Review by majormoses — APPROVED** [review]
LGTM

**Comment by majormoses** [comment]
released: https://supermarket.chef.io/cookbooks/netuitive/versions/0.21.0

**Pull request merged**